tests/test_main.py

```diff
@@ -0,0 +1,125 @@
+"""Run from the `pyta` root directory to use the local `python_ta` rather than
+installed `python_ta` package.
+"""
+
+import subprocess
+from os import environ, path
+from sys import executable as sys_executable
```

Comment:
@quiz3 Please change this back. Don't make unnecessary changes to get tests to re-run. You can message me on Slack and I'll re-run them.